README.org

```diff
@@ -52,23 +52,33 @@ ones above:
-| desktop-environment-brightness-decrement-slowly | ~S-<XF86MonBrightnessDown>~ |
-| desktop-environment-volume-increment            | ~<XF86AudioRaiseVolume>~    |
-| desktop-environment-volume-decrement            | ~<XF86AudioLowerVolume>~    |
-| desktop-environment-volume-increment-slowly     | ~S-<XF86AudioRaiseVolume>~  |
-| desktop-environment-volume-decrement-slowly     | ~S-<XF86AudioLowerVolume>~  |
-| desktop-environment-toggle-mute                 | ~<XF86AudioMute>~           |
-| desktop-environment-toggle-microphone-mute      | ~<XF86AudioMicMute>~        |
-| desktop-environment-screenshot-part             | ~S-<print>~                 |
-| desktop-environment-screenshot                  | ~<print>~                   |
-| desktop-environment-lock-screen                 | ~s-l~                       |
-| desktop-environment-toggle-wifi                 | ~<XF86WLAN>~                |
-| desktop-environment-toggle-bluetooth            | ~<XF86Bluetooth>~           |
+| *Command*                                          | *Key binding*               |
+|----------------------------------------------------+-----------------------------|
+| desktop-environment-brightness-increment           | ~<XF86MonBrightnessUp>~     |
+| desktop-environment-brightness-decrement           | ~<XF86MonBrightnessDown>~   |
+| desktop-environment-brightness-increment-slowly    | ~S-<XF86MonBrightnessUp>~   |
+| desktop-environment-brightness-decrement-slowly    | ~S-<XF86MonBrightnessDown>~ |
+| desktop-environment-volume-increment               | ~<XF86AudioRaiseVolume>~    |
+| desktop-environment-volume-decrement               | ~<XF86AudioLowerVolume>~    |
+| desktop-environment-volume-increment-slowly        | ~S-<XF86AudioRaiseVolume>~  |
+| desktop-environment-volume-decrement-slowly        | ~S-<XF86AudioLowerVolume>~  |
+| desktop-environment-toggle-mute                    | ~<XF86AudioMute>~           |
+| desktop-environment-toggle-microphone-mute         | ~<XF86AudioMicMute>~        |
+| desktop-environment-screenshot-part                | ~S-<print>~                 |
+| desktop-environment-screenshot                     | ~<print>~                   |
+| desktop-environment-lock-screen                    | ~s-l~                       |
+| desktop-environment-toggle-wifi                    | ~<XF86WLAN>~                |
+| desktop-environment-toggle-bluetooth               | ~<XF86Bluetooth>~           |
+| desktop-environment-keyboard-layout-cycle-forward  | ~s-SPC~                     |
+| desktop-environment-keyboard-layout-cycle-backward | ~s-S-SPC~                   |
+
+*** Keyboard layouts
+
+Keyboard layouts are stored in a variable called ~desktop-environment-keyboard-layouts~ as parameters for the ~setxkbmap~ shell command. (To be accurate ~setxkbmap -layout ~) To load keyboard layout automatically on startup add following to your configuration file:
```

Comment:
I would say `setxkbmap -layout` immediately because our user base is quite geeky already :-).